istioctl/cmd/istioctl/handlers.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"strings"
+)
+
+func getProxyDetails(proxyName string) (string, string, error) {
```

Comment:
@liamawhite need help regarding test case for this, this depends on "handleNamespaces", which depends on current commands namespace or current-conxtext. 

Can you give some pointers for this !

Reply:
The tests should be in scope so just set the namespace variable.

Reply:
Thanks @liamawhite for your help and mentoring.